boefjes/boefjes/plugins/kat_dns_version/normalize.py

```diff
@@ -0,0 +1,108 @@
+import json
+from collections.abc import Iterable
+
+from boefjes.job_models import NormalizerOutput
+from octopoes.models import Reference
+from octopoes.models.ooi.findings import CVEFindingType, Finding
+from octopoes.models.ooi.software import Software, SoftwareInstance
+from packaging.version import Version
+
+cves = {
```

Comment:
If used as a constant
```suggestion
CVES = {
```